pysb/bng.py

```diff
@@ -103,12 +104,16 @@ class BngBaseInterface(object):
     @abc.abstractmethod
     def __init__(self, model, verbose=False, cleanup=False,
                  output_prefix=None, output_dir=None):
+        self._base_file_stem = 'pysb'
         self.verbose = verbose
         self.cleanup = cleanup
         self.output_prefix = 'tmpBNG' if output_prefix is None else \
             output_prefix
-        self.generator = BngGenerator(model)
-        self._check_model()
+        if model:
```

Comment:
A limited one: using the BNG interface with a model loaded directly from a BNGL file, as I've done in the new unit tests for this PR in pysb/tests/test_importers.py. It does complicate the code slightly but the added functionality is useful, unless I'm missing another way of achieving this.